website/src/components/drawer.tsx

```diff
@@ -0,0 +1,91 @@
+"use client";
+
+import { useOnClickOutside } from "hooks/useOnClickOutside";
+import { useOnKeyUp } from "hooks/useOnKeyUp";
+import { ComponentType } from "react";
+import ReactDOM from "react-dom";
+
+import { CloseIcon } from "./icons/closeIcon";
+
+type Props = {
+  children: React.ReactNode;
+  container?: HTMLElement;
+  onClose?: () => void;
```

Comment:
This is basically nitpick 😆 but I like to use VoidFunction.. 
```suggestion
  onClose?: VoidFunction;
```